broker-client/src/test/java/io/camunda/zeebe/broker/client/api/BrokerTopologyManagerTest.java

```diff
@@ -419,6 +421,58 @@ void shouldNotOverwriteClusterSizeFromBrokerInfo() {
     assertThat(topologyManager.getTopology().getClusterSize()).isEqualTo(2);
   }
 
+  @Test
+  void shouldUpdatePartitionsCountFromClusterTopology() {
```

Comment:
:upside_down_face: Didn't remember this was implemented. Thanks for adding the tests.